RoboJackets/igvc-software
# Feat/swerve sim

# Description
This PR adds a Swervi robot simulation. The robot consists of 4 submodules, each with its own swivel and wheel. The robot also has a Lidar sensor but is missing cameras, IMUs, and the GPS sensor, as the specific position of these sensors is yet to be determined by the mechanical subteam. Those can be easily added later.

This PR does the following:
- Adds a Swervi model that can be added to future simulations.

Fixes #721 

# Testing steps (If relevant)
1. run `roslaunch igvc_description simulationTesting.launch`
2. observe the robot in Gazebo and make sure all the joints rotate properly when applying forces to them (this can be done by right-clicking and selecting Apply Force/Torque). 

Expectation: The thing which does the thing does the thing

# Self Checklist
